tests/pc/test_po_update.py

```diff
@@ -149,6 +149,8 @@ def test_po_update_io_no_loss(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
     # THEN no packets shall loss
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
     asichost = duthost.asic_instance(enum_frontend_asic_index)
+    namespace = asichost.namespace
+    is_single_asic = not namespace
     mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
 
```

Comment:
```
        mg_facts = duthost.asic_instance(
            enum_frontend_asic_index
        ).get_extended_minigraph_facts(tbinfo)
```

Reply:
Thanks for letting me know this way, refined